code/modules/admin/gimmick/wizard_rings.dm

```diff
@@ -58,15 +58,12 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves/ring/wizard)
 
 		unequipped(var/mob/user)
 			..()
-			var/show_message = 0
-			if (user?.bioHolder.HasEffect("telekinesis"))
-				user.bioHolder.RemoveEffect("telekinesis")
-				show_message = 1
-			if (user?.bioHolder.HasEffect("hulk"))
+			if (user && user.bioHolder.HasEffect("hulk") && HAS_MOB_PROPERTY(user, PROP_PASSIVE_WRESTLE) && HAS_MOB_PROPERTY(user, PROP_STAMINA_REGEN_BONUS))
+				REMOVE_MOB_PROPERTY(user, PROP_PASSIVE_WRESTLE, "empower")
+				REMOVE_MOB_PROPERTY(user, PROP_STAMINA_REGEN_BONUS, "empower")
```

Comment:
I don't think you need to check for existence of the props when removing, and they shouldn't share a conditional with the hulk check anyways

Reply:
Normally I don't but from what I got they only wanted the message to show up when it acctually got removed, could always just remove the message or only check for hulk.